### PR TITLE
fix: Modified flag descriptions to eliminate new line indentation issue

### PR DESCRIPTION
**Pull request by shrutiburman**
Fixes [DII-132](https://issues.corp.twilio.com/browse/DII-132)

This PR: 
Solves the [indentation issue](https://github.com/jamestalmage/supports-hyperlinks/issues/13) introduced via dependency supports-hyperlink (to convert markdowns into url).
Alternative dependencies aren't compliant with node 10.x. 

Before: 
Screenshot 2021-11-08 at 10 54 03 AM

After:
Screenshot 2021-11-08 at 10 54 47 AM

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [ ] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guidelines](https://github.com/twilio/twilio-cli-core/blob/main/CONTRIBUTING.md) and my PR follows them
- [x] I have titled the PR appropriately
- [ ] I have updated my branch with the main branch
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added the necessary documentation about the functionality in the appropriate .md file
- [ ] I have added inline documentation to the code I modified

If you have questions, please file a [support ticket](https://twilio.com/help/contact), or create a GitHub Issue in this repository.

### CODE REVIEW

**Review by shamantraghav — APPROVED** [review]

File: src/services/twilio-api/twilio_api_v2010.json [hunk 1]
```diff
@@ -10597,7 +10597,7 @@
             }
           },
           {
-            "description": "The pattern on which to match phone numbers. Valid characters are `*`, `0-9`, `a-z`, and `A-Z`. The `*` character matches any single digit. For examples, see [Example 2](https://www.twilio.com/docs/phone-numbers/api/availablephonenumberlocal-resource?code-sample=code-find-phone-numbers-by-number-pattern) and [Example 3](https://www.twilio.com/docs/phone-numbers/api/availablephonenumberlocal-resource?code-sample=code-find-phone-numbers-by-character-pattern). If specified, this value must have at least two characters.",
+            "description": "The pattern on which to match phone numbers. Valid characters are `*`, `0-9`, `a-z`, and `A-Z`. The `*` character matches any single digit. If specified, this value must have at least two characters. See [Examples](https://www.twilio.com/docs/phone-numbers/api/availablephonenumberlocal-resource?code-sample=code-find-phone-numbers-by-number-pattern)",
```
Comment (ravali-rimmalapudi) [inline src/services/twilio-api/twilio_api_v2010.json:10600]: These are generated files right, If we change something here will it not overwrite in future by` api-definitions`?

Comment (shrutiburman) [reply src/services/twilio-api/twilio_api_v2010.json:10600]: Yes possible, there's no description field in api-definitions as such and the basic [json](https://code.hq.twilio.com/twilio/api-definitions/tree/603e2cf5e50d08c55c465654c14f1d0309de21f0/base/autopilot/v1) hasn't been modified a lot, last modified was 3 yrs ago. I tried finding the source of `description` and found [changes](https://github.com/twilio/twilio-cli-core/pull/117) on this repo itself to include it.

Comment (ravali-rimmalapudi) [reply src/services/twilio-api/twilio_api_v2010.json:10600]: Okay. Then It's fine.

**Review by ravali-rimmalapudi — APPROVED** [review]
Looks Good.

File: src/services/twilio-api/twilio_api_v2010.json (hunk 1 above)
Comment (shrutiburman) [reply src/services/twilio-api/twilio_api_v2010.json:10600]: Thanks for confirming @shwetha-manvinkurke, I'll raise a PR for these changes on api-definitions.

**Pull request merged**